linux_docker_resources/Dockerfile

```diff
@@ -89,7 +89,18 @@ RUN echo "@today_str"
 RUN if test \( ${BRIDGE} = true -o ${INSTALL_TURTLEBOT2_DEMO_DEPS} = true \) ; then apt-get update && apt-get install --no-install-recommends -y ros-melodic-catkin; fi
 
 # Install build and test dependencies of ros1_bridge.
-RUN if test ${BRIDGE} = true; then apt-get update && apt-get install --no-install-recommends -y python-rospkg ros-melodic-common-msgs ros-melodic-rosbash ros-melodic-roscpp ros-melodic-roslaunch ros-melodic-rosmsg ros-melodic-roscpp-tutorials ros-melodic-rospy-tutorials ros-melodic-tf2-msgs; fi
+RUN if test ${BRIDGE} = true; then apt-get update && apt-get install --no-install-recommends -y \
+    python-rospkg \
+    python3-crypto \
+    python3-gnupg \
```

Comment:
I'm curious why we bring in the dependencies manually here instead of installing the ros package that needs them?

Reply:
the ros package installs the python 2 version of these dependencies but for some reason I needed the python3 version for this to work in my container.

Currently, while installing the python3 version works locally, neither work on CI so I need to find another solution for the `Crypto` module to be found properly